jobsdb/jobsdb.go

```diff
@@ -235,6 +235,9 @@ type JobsDB interface {
 	//    })
 	StoreWithRetryEachInTx(ctx context.Context, tx StoreSafeTx, jobList []*JobT) (map[uuid.UUID]string, error)
```

Comment:
`StoreWithRetryEachInTx` & `StoreWithRetryEach` are being replaced by `StoreEachBatchRetryInTx`? Why aren't we removing the previoue methods?

Reply:
why didn't we keep the old methods and change their signature, since their only purpose was to serve gateway anyway :)

Reply:
can be removed yeah.
I will 👍